src/System.IO.IsolatedStorage/src/project.json

```diff
@@ -8,13 +8,16 @@
         "System.Diagnostics.Tools": "4.3.0-beta-24604-02",
         "System.IO": "4.3.0-beta-24604-02",
         "System.IO.FileSystem": "4.3.0-beta-24604-02",
+        "System.IO.FileSystem.AccessControl": "4.3.0-beta-24604-02",
```

Comment:
> This is going to cause this extension library to be pulled into the shared framework on both windows and linux which is unfortunate. How hard would this be to write as a raw PInvoke?

For the Windows and Linux part, we have the ability to specialize the dependencies based on platform, right?

Regarding using P/Invokes directly, Is your thinking that the P/Invokes we'd need and the code related to invoking them would be significantly/meaningfully smaller than the AccessControl library?